app/controllers/concerns/idv/doc_auth_vendor_concern.rb

```diff
@@ -6,7 +6,7 @@ module DocAuthVendorConcern
 
     # @returns[String] String identifying the vendor to use for doc auth.
     def doc_auth_vendor
-      if resolved_authn_context_result.facial_match?
+      if resolved_authn_context_result.facial_match? || Idv::SocureUserSet.new.maxed_users?
```

Comment:
may or may not be an issue here but similarly to how we use RateLimter, could this be optimized not create a new instances and reuse  connections
```suggestion
      if resolved_authn_context_result.facial_match? || socure_user_set.maxed_users?
```

def socure_user_set
    @socure_user_set ||= SocureUserSet.new
end